src/jlpkgenv.ts

```diff
@@ -196,6 +202,12 @@ function absEnvPath(p: string) {
     }
 }
 
+function parseEnvVariables(p: string) {
+    return p.replace(/\${env:(.*?)}/g, function (variable) {
+        return process.env[variable.match(/\${env:(.*?)}/)[1]] || ''
```

Comment:
```suggestion
    return p.replace(/\${env:(.*?)}/g, (_, variable) => {
        return process.env[variable] || ''
```
No need to apply the regex twice.